Vagrantfile

```diff
@@ -12,6 +12,9 @@ Vagrant.configure("2") do |config|
   # => Makes it two-way sync
   config.vm.synced_folder ".", "/vagrant", type: "virtualbox"
 
+  # Shortcuts used later inside dev vm
+  config.vm.provision "file", source: "aliases.sh", destination: "~/.aliases.sh"
```

Comment:
This sets up the aliases for the root account and for the duration of the `aliases.sh` run - Probably no what you want. To properly set aliases you must have them defined from a shell startup file such as `.bashrc` (You can have `.bashrc` source another script if you want)

Reply:
It set's up fpr the vagrant user, not root. Verified.
However maybe implicitly stating /home/vagrant is more appropriate. Will change

Reply:
Oh I think I misunderstood what you're doing here - so this is installing the file, not running it? You may want to alter your comment above to make that clearer.

Also, maybe just source the file from `/vagrant` instead of explicitly copying it to another location?